core/modules/system/system.module

```diff
@@ -4660,3 +4666,315 @@ function system_transliterate_machine_name($string, $options = array()) {
   return $string;
 }
 
+/**
+ * Implements hook_process_html().
+ */
+function system_process_html(&$variables) {
+  backdrop_emit_cache_tags();
+  backdrop_emit_cache_max_age();
+}
+
+/**
+ * Implements hook_ajax_render_alter().
+ */
+function system_ajax_render_alter(&$commands) {
+  backdrop_emit_cache_tags();
+  backdrop_emit_cache_max_age();
+}
+
+/**
+ * Implements hook_emit_cache_tags().
+ */
+function system_emit_cache_tags($tags) {
+  $page_cache_tags = &backdrop_static(__FUNCTION__, array());
+
+  if (config_get('system.core.settings', 'emit_cache_tags')) {
```

Comment:
This should be `system.core` not `system.core.settings`. There are a few instances of this.